docs/topics/templates/document-attributes.adoc

```diff
@@ -144,3 +144,7 @@
 
 :docs-name: appdev.openshift.io
 :link-docs: https://appdev.openshift.io
+
+// stage environment URLs for Docs and Launcher
+:docs-stage: https://appdev.prod-preview.openshift.io/
+:launcher-stage: https://launch.prod-preview.openshift.io/
```

Comment:
I'd advocate for using `link-docs-stage` and `link-launcher-stage` to keep consistency with other attributes (and to sort-of create namespaces for the attributes), but that's not a hard requirement, and I'm willing to accept this if necessary.

Reply:
@tradej Makes a lot of sense, I'll change the attribute name:)